### PR TITLE
add eslint

### LINKED ISSUE #20: eslint

### PR DESCRIPTION
**Pull request by Nikzz3**

### CODE REVIEW

**Comment by Sl1van** [comment]
Everything seems fine except the package-lock.json got committed.

**Review by OG-Jons — CHANGES_REQUESTED** [review]
Remove package-lock.json / add it to .gitignore

File: src/App.jsx [hunk 1]
```diff
@@ -2,10 +2,11 @@ import logo from './logo.svg';
 import './App.css';
 import {Button} from "@material-ui/core";
 import {firebase} from "./server/firebaseConfig";
+import React from 'react';
```
Comment (OG-Jons) [inline src/App.jsx]: React import is not needed

Comment (Nikzz3) [reply src/App.jsx]: Without React import eslint throws error

Comment (OG-Jons) [reply src/App.jsx]: Then eslint has too old formatting, react isn't needed anymore

Comment (OG-Jons) [reply src/App.jsx]: https://stackoverflow.com/questions/44141010/do-i-need-to-import-react-for-stateless-functional-components
It's explained in detail here

File: .gitignore [hunk 2]
```diff
@@ -24,4 +24,5 @@ yarn-error.log*
 *.lock
 
 .idea/
-.vscode/
\ No newline at end of file
+.vscode/
+package-lock.json
```
Comment (OG-Jons) [inline .gitignore]: newline needed at the end of file

File: src/components/Example/Example.component.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 export default function Example() {
     return (
         <>Example for structure</>
-    )
+    );
 }
```
Comment (OG-Jons) [inline src/components/Example/Example.component.jsx]: newline here too

**Review by OG-Jons — APPROVED** [review]
still against eslint, but eh

**Comment by Sl1van** [comment]
👍

**Review by Sl1van — APPROVED** [review]
👍
everything seems fine

**Pull request merged**